source/actors/Gravity.gd

```diff
@@ -0,0 +1,9 @@
+extends "res://actors/PlatformPhysics.gd"
```

Comment:
why don't you use `class_name` keyword? it is simpler and more readable (imo)

Reply:
Ohh I forgot about it!

Yeah it automatically update the path of the class file, right? Gonna fix that. Thanks for the reminder!